src/features/supervisor-app/hooks/supervisor-app.ts

```diff
@@ -18,8 +17,10 @@ hooks.addPureHook('PATCH', 'resin', 'device', {
 	async PRERUN(args) {
 		if (args.request.values.supervisor_version != null) {
 			const ids = await sbvrUtils.getAffectedIds(args);
-			await setSupervisorReleaseResource(
-				args.api,
+			const api = args.api;
```

Comment:
This is purely stylistic, ignore if you want:

```suggestion
			const { api } = args;
```